recipes/siphon/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+    name: siphon
+    version: "0.3.2"
+
+source:
+    fn: siphon-0.3.2.tar.gz
+    url: https://pypi.python.org/packages/source/s/siphon/siphon-0.3.2.tar.gz
+    md5: dd08ff89b1176939d59acae0d544dd81
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - numpy
```

Comment:
Please correct me if I am wrong but siphon does not seems to have any compiled code against numpy so that is not necessary.

Note that we can add it anyways. The default channel does that for all package.  It would be done by adding `numpy x.x` as a dependency.  However, that has the downside of bloating the channel, by creating many packages like `pack-py{versions}-np{versions}`, and using more CI time than needed.

BTW the numpy version range is in the CI configuration. See https://github.com/conda-forge/staged-recipes/blob/master/scripts/run_docker_build.sh#L56 for an example.